.github/workflows/remark.yml

```diff
@@ -0,0 +1,29 @@
+name: Remark
+
+on:
+  push:
+    branches:
+      - trying
+      - staging
+      - master
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  remark:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Setup
+    - name: Checkout
+      uses: actions/checkout@v2.3.3
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v1.4.4
+
+    - name: Install remark
+      run: npm install remark-cli remark-lint remark-lint-maximum-line-length remark-preset-lint-recommended
+
+    # Run
+    - name: Check *.md files
+      run:  git ls-files  '*.md' | grep -v "gcc/config*" | grep -v "gcc/testsuite" | grep -v "gcc/common.md" | xargs -I {} ./node_modules/.bin/remark {} -u lint -f > /dev/null
```

Comment:
I think you don't need to have all these grep here, ls-files already behaves like ls:
```
$ git ls-files *.md
CODE_OF_CONDUCT.md
CONTRIBUTING.md
README.md
```

Reply:
I guess this could be changed to 
```
run:  git ls-files  '*.md'  | xargs -I {} ./node_modules/.bin/remark {} -u lint -f > /dev/null
```

Reply:
I put that exlucsion in to avoid linting the machine descriptions in GCC which are lisp. Though it does pass if you leave it like this which is interesting.

Reply:
I guess `ls-files` only list the root if not given any other args.
You would need something like `git ls-files **/**/**/*.md` to get the target md files.

Reply:
I've updated this now, lets see how it works. Since this automation is looking at the root may wish to extend this if we add mark-down docs within the rust dir's at some point.